src/Components/Bookshelf/Bookshelf.tsx

```diff
@@ -51,7 +51,7 @@ class Bookshelf extends React.Component<BookshelfProps, BookshelfState> {
     }
     return (
       !this.state.books ? <h3>Loading</h3>
-      : <div>{bookCards}</div>
+      : <div className='bookshelf'>{bookCards}</div>
```

Comment:
Nice, I've been getting in the habit of giving most things a className because of cypress.